route_unspecified.go

```diff
@@ -2,10 +2,38 @@
 
 package netlink
 
+import (
+	"net"
+)
+
 func (r *Route) ListFlags() []string {
 	return []string{}
 }
 
 func (n *NexthopInfo) ListFlags() []string {
 	return []string{}
 }
+
+func (h *Handle) RouteAdd(route *Route) error {
```

Comment:
Current code organization seems to want any non implemented `func (h *Handle)` method in `handle_unspecified.go`